lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -30,6 +30,25 @@ using namespace mlir::torch::Torch;
 // Utilities
 //===----------------------------------------------------------------------===//
 
+OpFoldResult genericViewLikeFold(Attribute self, Type resultType) {
+  constexpr int64_t kMaxFoldSize = 16;
```

Comment:
What's the rationale behind `kMaxFoldSize` to be 16?

Reply:
None, good catch.